generator/source/release/bumper/notes.md

```diff
@@ -0,0 +1,322 @@
+---
+layout: page
+hide: true
+title: Hyperledger Sawtooth 1.1 (Bumper)
+permalink: /release/bumper/
+release: 1.1
+release-name: bumper
+# Copyright (c) 2018 Bitwise IO, Inc.
+# Licensed under Creative Commons Attribution 4.0 International License
+# https://creativecommons.org/licenses/by/4.0/
+---
+
+<a href="top"></a>
+
+Hyperledger Sawtooth 1.1 (Bumper) is now available. See the [latest
+documentation](https://sawtooth.hyperledger.org/docs/core/releases/latest/) to
+get started.
+
+This page describes the new and changed features in Hyperledger
+Sawtooth 1.1 (since release 1.0). Information is organized in the following
+categories:
+
+- [Core Sawtooth Components](#core-sawtooth-components)
+- [Consensus](#consensus): Includes the new consensus API, changes to existing
+  consensus algorithm implementations, and support for new consensus algorithms
+- [Documentation](#documentation)
+- [Smart Contract Engines](#smart-contract-engines)
+- [DevOps and CI](#devops-and-ci): Describes changes to the project's build
+  tools and continuous integration process
+- [Website](#website): Includes the Sawtooth website content, structure, and
+  website development processes
+
+# Core Sawtooth Components
+
+## New Features
+
+- The consensus API has been completely redesigned. Consensus has been moved to
+  a separate process, called a “consensus engine”. See [Hyperledger Sawtooth
+  Consensus](#hyperledger-sawtooth-consensus) for more details.
```

Comment:
This is supposed to link with the "Consensus" section lower down.

`# Consensus`